.eslintrc.js

```diff
@@ -0,0 +1,67 @@
+module.exports = {
+    "env": {
+        "es6": true,
+        "browser": true,
+        "node": true,
+        "jquery": true,
+        "mocha": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": 6,
+        "sourceType": "module",
+        "ecmaFeatures": {
+          "jsx": true
+        }
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
```

Comment:
It would be nice to set a rule for the maximum length allowed per line in the eslint file.

Reply:
Done